zipline/errors.py

```diff
@@ -639,6 +639,16 @@ class NoSuchPipeline(ZiplineError, KeyError):
     )
 
 
+class DuplicatePipelineName(ZiplineError):
+    """
+    Raised when a user tries to attach a pipeline with a name that already
+    exists for another attached pipeline.
+    """
+    msg = (
+        "Pipeline name '{name}' already exists for another pipeline."
```

Comment:
You can also use `{name!r}` here instead of having quotes around the format unit.

Reply:
Might be helpful to include a suggestion of how to fix the issue here, e.g. "Tried to attach a pipeline named {name}, but there was already an attached pipeline with that name. Please use a different name for this pipeline."